### DISCUSSION TITLE
feat: Add `TRACE_EVENTS` flag

### DISCUSSION THREAD

**Discussion by JorisAndrade** [langfuse/langfuse#1716] [Ideas]
### Describe the feature or potential improvement

The `events` can be quite large on heavy setups while bringing limited value to most Langfuse users.

A `TRACE_EVENTS` flag (that can default to `true`) could be added to turn off events and allow users to turn in back on for debugging purposes if they have to.

**Comment by marcklingen** [comment]
Great idea, added an issue for this: https://github.com/langfuse/langfuse/issues/1717

**Comment by marcklingen** [comment]
Released in https://github.com/langfuse/langfuse/releases/tag/v2.36.0
